pandas/tests/frame/test_arithmetic.py

```diff
@@ -111,6 +111,8 @@ def test_df_radd_str(self):
 
 class TestPeriodFrameArithmetic(object):
 
+    @pytest.mark.xfail(reason="Unary negative is currently not permitted "
```

Comment:
@deniederhut so period - period is now going to raise? this is https://github.com/pandas-dev/pandas/issues/13077.

I am not actually sure what we should do here. I think we can re-enable these though, at least for now.

(you can use ``is_period_arraylike`` to detect this)

Reply:
The test is marked xfail because of the `-exp` in this assertion `tm.assert_frame_equal(df - p, -exp)`, not the initial subtraction. Happy to allow it and unmark the tests.

Should the expected return values in the tests be scalars? I think `period` -  `period` currently returns an integer.

Reply:
i c
the return value should be integers